### ISSUE TITLE
[ERROR] Exception during symbol loading - 'Key '2905' not present in dictionary'

### ISSUE THREAD

**Issue by Ayymoss** [xoofx/ultra#14]
During runtime, I'm running into an exception when loading symbols. Sometimes it fails early or towards the end - but always on the same stage. The key ID has been different each run.

[Ultra-Runtime-Exception.txt](https://github.com/user-attachments/files/17959725/Ultra-Runtime-Exception.txt)

If needed, I can provide the application I'm trying to profile.

```
PS C:\Windows\system32> dotnet --list-sdks
8.0.401 [C:\Program Files\dotnet\sdk]
9.0.100 [C:\Program Files\dotnet\sdk]
```

**Comment by xoofx** [comment]
Just to make sure, which version exactly do you run?

**Comment by Ayymoss** [comment]
I installed it today, ref Nick Chapsas' video, so the latest (I believe) on NuGet. 

PS C:\Windows\system32> ultra --version
2.0.0+20b85170ee37a839bf2e1538f004651f4615f00f

**Comment by xoofx** [comment]
Thanks, I will try to investigate this later tonight.

**Comment by xoofx** [comment]
I have made a tentative fix via 225fdc1 available in `2.0.1+`

Please try it, hopefully it should fix your issue. I'm still a bit unclear how that could happen in the first place, but now the code should be more correct. Let's see 🤞
